index.js

```diff
@@ -46,12 +53,12 @@ function attachToBody (options, req, reply, next) {
   })
 }
 
-function defaultConsumer (field, file, filename, encoding, mimetype, body) {
+function defaultConsumer (field, file, filename, encoding, mimetype, body, fileCount) {
```

Comment:
`fileCount` should be called `fileIndex` here.

Reply:
done